app/components-react/highlighter/ExportModal.tsx

```diff
@@ -74,7 +86,7 @@ export default function ExportModal(p: { close: () => void }) {
 }
 
 function ExportProgress() {
-  const { exportInfo, cancelExport } = useModule(ExportModule);
+  const { exportInfo, cancelExport } = useModule(ExportController);
```

Comment:
Is it intentional that this wasn't replaced with `useController` and passing in the context instead?

Reply:
Good catch